sunpy/timeseries/timeseries_factory.py

```diff
@@ -287,7 +289,11 @@ def _parse_args(self, *args, **kwargs):
                                 # to preserve multi-line comments
                                 if isinstance(args[i+1], astropy.io.fits.header.Header):
                                     args[i+1] = MetaDict(sunpy.io.header.FileHeader(args[i+1]))
-                                meta.update(args[i+1])
```

Comment:
This is still needed for the case when header is passed no?

Reply:
It is needed and I have retained it. It is part of the ``else`` condition in case the ``meta`` isn't a ``TimeSeriesMeta`` as it would have been without the proposed modification as proposed in this PR.

Reply:
oh sorry of course - my bad!